src/Integration.Vsix/CFamily/VcxProject/FileConfig.cs

```diff
@@ -0,0 +1,283 @@
+﻿/*
```

Comment:
SC is complaining about a whitespace difference -- I fixed it and will push along with PR fixes.